server/lib/accounts.js

```diff
@@ -91,6 +116,23 @@ Accounts.onCreateUser(function(options, user = {}) {
 		}
 	}
 
+	if (!user.active) {
+		user.emails.some((email) => {
+			RocketChat.models.Roles.findUsersInRole('admin').forEach(function(adminUser) {
+				email = {
+					to: adminUser.emails[0].address,
+					from: RocketChat.settings.get('From_Email'),
+					subject: Accounts.emailTemplates.notifyAdmin.subject(),
+					html: Accounts.emailTemplates.notifyAdmin.html(user)
+				};
+
```

Comment:
Instead of sending an e-mail for each of the admins, I'd make a list of admin emails and send only one e-mail with everyone on it. 

One of the advantages to this is knowing who are the other admins that received that email and might have acted on it before you do.